Kitodo/src/test/java/org/kitodo/production/exporter/download/ExportMetsIT.java

```diff
@@ -60,7 +60,7 @@ public static void setUp() throws Exception {
         exportUri = ConfigCore.getUriParameter(ParameterCore.DIR_USERS, userDirectory);
 
         fileService.createDirectory(URI.create(""), metadataDirectory);
-        fileService.copyFile(URI.create("testmetaNewFormat.xml"), URI.create(metadataDirectory + "/meta.xml"));
+        fileService.copyFile(URI.create("metadataFiles/testmetaNewFormat.xml"), URI.create(metadataDirectory + "/meta.xml"));
```

Comment:
Is it possible to generalize the use of paths, for example, by adding them to the base test class or by using a utility class?